test/async.js

```diff
@@ -106,32 +107,24 @@ test('throw an Error if `src` does not exists', async t => {
 	t.regex(err.stack, /`NO_ENTRY`/, err);
 });
 
-test('rethrow mkdir EACCES errors', async t => {
-	const mkdir = fs.mkdir;
+test.serial('rethrow mkdir EACCES errors', async t => {
```

Comment:
I would do `import { serial as test } from 'ava';` and keep `test.serial` as is. Actually, because you stub different properties in the tests, you could keep running them async, no?

Reply:
Do we want all of them to run in serial though? Or am I misunderstanding you here?

> Actually, because you stub different properties in the tests, you could keep running them async, no?

No, if some method isn't restored when running another test, it might throw in that test.

Reply:
Oops, I totally missed all the other tests. Was looking at it on the train with bad internet connection...

> No, if some method isn't restored when running another test, it might throw in that test.

You're right, I thought these where the only tests, missed the others, my bad :).